teamd/teamd_runner_ttdp.c

```diff
@@ -445,7 +445,7 @@ static uint8_t update_aggregate_state(struct teamd_context *ctx,
 			if (!inhibit) {
 				/* if a node was discovered while we were gone, we need to become a
 				 * middle node */
```

Comment:
Consider updating this comment to also describe the current case.

Reply:
Inaccurate comment removed.